### PR TITLE
Turkish Translations updated

### PR DESCRIPTION
**Pull request by Ardakilic**
This pull request includes missing translations for Turkish language, also includes retouches and some grammatical errors included in the translation.

I have checked almost every string and made necessary edits.

Edit: Also, please squash if this will be merged :)

### CODE REVIEW

File: CliClient/locales/tr_TR.po [hunk 1]
```diff
@@ -3374,126 +3381,4 @@ msgstr "OneDrive ile giriş yapın"
...
-
-#~ msgid "Separate each tag by a comma."
-#~ msgstr "Her etiketi virgülle ayırın."
-
-#~ msgid ""
-#~ "The path to synchronise with when file system synchronisation is enabled. "
-#~ "See `sync.target`."
-#~ msgstr ""
-#~ "Dosya sistemi senkronizasyonu etkinleştirildiğinde senkronize edilecek "
-#~ "yol. Bakınız `sync.target`."
```
Comment (tessus) [inline CliClient/locales/tr_TR.po]: Please add these back. There's a reason why we keep old translations around.

**Comment by tessus** [comment]
I can merge as soon as the removed translations are back in and the change (commit) to the README.md is removed.

**Comment by tessus** [comment]
P.S.: We always `Squash and merge`.

File: CliClient/locales/tr_TR.po (hunk 1 above)
Comment (Ardakilic) [reply CliClient/locales/tr_TR.po]: My translation tool deleted them apparently. Adding them back now.

Comment (tessus) [reply CliClient/locales/tr_TR.po]: Ah, I see. I think there's an option in Poedit. Let me check.

**Comment by Ardakilic** [comment]
All should be OK now, hopefully. Reverted the README and added the commented old translations back.

File: CliClient/locales/tr_TR.po (hunk 1 above)
Comment (tessus) [reply CliClient/locales/tr_TR.po]: Nope, couldn't find a setting in Poedit. I must have something else in mind.

**Comment by tessus** [comment]
Great, thank you for this extensive update!

**Comment by Ardakilic** [comment]
Awesome, thanks for the review and the merge!

**Pull request merged**